tests/src/contracts.rs

```diff
@@ -0,0 +1,23 @@
+use cw_orch::{Uploadable,WasmPath,CwEnv,Contract,Daemon,contract, TxHandler};
+
+const CRATE_PATH: &str = env!("CARGO_MANIFEST_DIR");
+
+#[contract(abstract_core::ans_host::InstantiateMsg,abstract_core::ans_host::ExecuteMsg,abstract_core::ans_host::QueryMsg,abstract_core::ans_host::MigrateMsg)]
+pub struct ANSHost;
```

Comment:
Is this here because the the other interfaces still use boot?

Reply:
Oh no I forgot to remove this code, I was experimenting on

Reply:
removed in https://github.com/AbstractSDK/contracts/pull/329/commits/8324ad841b8158b26dc1e5cb298c4bceb42a1bff